float64 speed_percentage
---